ibis/pyspark/compiler.py

```diff
@@ -123,6 +126,35 @@ def compile_self_reference(t, expr, scope, **kwargs):
     return t.translate(op.table, scope)
 
 
+@compiles(ops.Cast)
+def compile_cast(t, expr, scope, **kwargs):
```

Comment:
Is this a partial implementation of cast that only supports casting literal to interval?

Reply:
This should support all general cast operations that can be done with `src_column.cast(type_str)` in PySpark. It's possible there are other edge cases like casting literal to interval, but it should work for the normal case. I'll add a `test_cast()` to Pyspark backend `test_basic.py`.